fork-clone-workflow.md

```diff
@@ -33,9 +35,61 @@ own version of the https://github.com/mindfulness-at-the-computer/git-info reosi
 username is `joeuser`, the URL for your version of the repository will be
 https://github.com/joeuser/git-info.
 
-## Step 2 - Make a local clone of your fork
+## Setup Step 2 - Make a local clone of your fork
 This step is perfomed on the development system you will use to work on the source code 
-for the project. 
-### Work in progress (taking a snapshot of work on 2/27/18)
+for the project. You will [clone](git-glossary.md#clone) your fork on to your development machine. 
+Then you will configure your clone so that it knows how to find the main (organizational) repository
+from which the fork was created.
+
+* Change directory to the location on your development machine where you want to store the clone. You can clone
+to any location, but it is a good idea to keep your clones in a fixed location. In this example `joeuser` keeps
+cloned repositories in the `/opt/repos` directory.
+```bash
+cd /opt/repos
+```
+* Clone your fork using the `git clone` command. You can find the URL of your fork on the main [gihub](git-glossary.md#github)
+page of your fork. Click on the `Clone or download` button and a popup will show you the URL to use if you want to
+`Clone with SSH`. The SSH option is the easiest option to use but requires that you have confiured configured your
+[github](git-glossary.md#github) account with your public SSH key. As an alternative, you may select `Use HTTPS`. 
+The HTTPS option uses a different URL and will require you to supply your [github](git-glossary.md#github)
+account password each time you access your fork. The SSH option does not require you to supply your [gihub](git-glossary.md#github)
+when accessing your fork.
+```bash
+git clone git@github.com:joeuser/git-info.git
+```
+* After executing the `git clone` command successfully, there will be a directory containing a clone of your fork.
+```bash
+$ ls -d git-info
+git-info
+```
+* Change directory to the newly create reporitory directory (`git-info` in this example) to configure the local clone so that it
+knows where to find the remote repository from which your fork was created. Your git clone maintains a set of
+[references](git_glossary.md#references) to remote repositories. 
+git automatically creates the `origin` reference when the clone is created. The `git remote -v` command lists the known
+[references](git_glossary.md#references) and the values associated with each reference.
+```bash
+$ cd git-info
+$ git remote -v
+origin  git@github.com:joeuser/git-info.git (fetch)
+origin  git@github.com:joeuser/git-info.git (push)
+```
+* By convention we use a reference named `upstream` to point to the remote repository from which your fork was 
```

Comment:
"By convention we use a reference named `upstream` to point to the remote repository" --- can we change this to "Following convention ..." or something like this?

Reply:
Yes, I will make the change.